tests/container_tests/test_zip_container.py

```diff
@@ -254,6 +257,14 @@ def test_nested_zip(self):
                     self.assertEqual(f.read(), self.nested_test_string)
 
     def test_stat(self):
-        # pass for now
-        # TODO(tianqi) add test after we well defined the stat
-        pass
```

Comment:
Would you mind adding a test to throw `NotADirectoryError` ?

Reply:
It is a test for `stat`, which returns the status of the file or directory, so I do not think the function should throw a `NotADirectoryError`. I will add another test for directory instead.